test/react-native/features/handled.feature

```diff
@@ -1,13 +1,13 @@
 Feature: Reporting handled errors
 
 Scenario: Calling notify() with a caught Error
-  When I run "HandledCaughtErrorScenario"
+  When I run "HandledJsErrorScenario"
   Then I wait to receive a request
   And the exception "errorClass" equals "Error"
-  And the exception "message" equals "HandledCaughtError"
+  And the exception "message" equals "HandledJsErrorScenario"
```

Comment:
Bordering on repetitive, perhaps, but we could add checked on "unhandled" in this file and `unhandled.feature` too.